pkg/mimirtool/analyze/grafana_test.go

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/grafana/cortex-tools/blob/main/pkg/analyse/grafana.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+
+package analyze
+
+import (
+	"testing"
+)
+
+func TestParseQuery(t *testing.T) {
+	tests := []struct {
+		query           string
+		expectedMetrics map[string]struct{}
+		shouldError     bool
+		errorMsg        string
```

Comment:
Given `shouldError` is always `false` and `errorMsg` is always `""`, are these necessary?

Or are there error cases that should be included?